maintenance.md

```diff
@@ -345,9 +345,8 @@ This section applies to the following containers. These containers are maintaine
 - [Contour Compatibility Matrix](https://projectcontour.io/resources/compatibility-matrix/)
 
 1. Check the [release page](https://github.com/envoyproxy/envoy/releases).
-2. Check the required bazel version in the following file.
-   - https://github.com/envoyproxy/envoy/blob/vX.Y.Z/.bazelversion
-3. Update the `version` and `bazel-version` arguments on the `build-envoy` job in the CircleCI `main` workflow.
+2. Update `version` of the `build_envoy` job in [`.github/workflows/main.yaml`](/.github/workflows/main.yaml).
+3. Update `clang_archive_path` in the same file if you want to update the clang version.
```

Comment:
```suggestion
3. Update `clang_archive_path` in [`.github/workflows/main.yaml`](/.github/workflows/main.yaml) if you want to update the clang version.
```

nit: I think it would be better to write the filename to avoid mistakes.